test/helper.js

```diff
@@ -268,6 +268,7 @@ module.exports.payloadMethod = function (method, t) {
       // Node errors for OPTIONS requests with a stream body and no Content-Length header
       if (upMethod !== 'OPTIONS') {
         var chunk = Buffer.allocUnsafe(1024 * 1024 + 1)
+        chunk.fill(0)
```

Comment:
why this change?

Reply:
Because when we set `asString`, string length will be less or equal to the buffer length since some character consists of multi buffer.
This make `receivedLength <= limit` when we allocate random buffer.

In this test suite, I think the goal was to make sure every character consist of one buffer, so the `receivedLentgh` is controllable to tes the body limit feature.

Reply:
Can you use `Buffer.alloc` instead?

Reply:
sure, done